public/js/views/course-display.js

```diff
@@ -0,0 +1,105 @@
+window.onload = function displayResults(){
```

Comment:
This is a better way to do something when the DOM is loaded. It's a jQuery shortcut for `$(document).on('ready', function() {});`. `window.onload` will happen later.
```js
$(function() {
  // code to be run when the DOM is loaded
});